qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -0,0 +1,182 @@
+- pci_bridge: install setup image_copy unattended_install.cdrom
+    type = pci_bridge
+    virt_test_type = qemu libvirt
+    login_timeout = 360
+    dd_test_cmd = "dd if=%s of=/dev/null bs=1k count=1000 iflag=direct &&"
+    dd_test_cmd += " dd if=/dev/zero of=%s bs=1k count=1000 oflag=direct"
+    variants:
+#        - nested_bridge:
+#        - illegal:
+#        - seven_bridges:
+#            variants:
+#                - hotplug_unplug:
+#                - migration:
+        - one_bridge:
+            pci_controllers +=" pci_bridge1"
+            type_pci_bridge1 = pci-bridge
+            variants:
+#                - hotplug_unplug:
+#                - migration:
+                - block_stress:
+                    only Linux
+                    type = pci_bridge_block_stress
+                    image_num = 10
+                    images += " stg0 stg1 stg2 stg3 stg4 stg5 stg6 stg7 stg8 stg9"
+#                    images += " stg10 stg11 stg12 stg13 stg14 stg15 stg16 stg17 stg18 stg19"
+#                    images += " stg20 stg21 stg22 stg23 stg24 stg25 stg26 stg27 stg28 stg29"
+#                    images += " stg30"
+                    disk_pci_bus_stg0 = pci_bridge1
+                    disk_pci_bus_stg1 = pci_bridge1
+                    disk_pci_bus_stg2 = pci_bridge1
+                    disk_pci_bus_stg3 = pci_bridge1
+                    disk_pci_bus_stg4 = pci_bridge1
+                    disk_pci_bus_stg5 = pci_bridge1
+                    disk_pci_bus_stg6 = pci_bridge1
+                    disk_pci_bus_stg7 = pci_bridge1
+                    disk_pci_bus_stg8 = pci_bridge1
```

Comment:
if most of your disk_pci_bus is pci_bridge, please write your configuration like:

disk_pci_bus = pci_bridge1

```
disk_pci_bus_image1 = pci.0
force_create_image = yes
force_create_image_image1 = no
image_size = 100M
```